### PR TITLE
[NavBar] Fix search-on-enter, SSR query persistence, missing admin / CMS dropdown links

### PR DESCRIPTION
**Pull request by damassi**
Addresses https://artsyproduct.atlassian.net/browse/DISCO-968
Addresses https://artsyproduct.atlassian.net/browse/DISCO-925

It was noticed today that search wasn't submitting on enter, due to mismatch described in 968 and overlooked during NavBar rebuild. Along the way I noticed 925 and fixed it, which involves search bar contents not being preserved from the server render to client-side mount pass, resulting in a UX flicker.

Companion PR: https://github.com/artsy/force/pull/4149

### CODE REVIEW

File: src/Apps/Artwork/Components/ArtworkBanner/index.tsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ export const ArtworkBannerFragmentContainer = createFragmentContainer(
           ... on ArtworkContextFair {
             name
             href
-            is_active
+            isActive
```
Comment (damassi) [inline src/Apps/Artwork/Components/ArtworkBanner/index.tsx]: Fixes a graphql deprecation, picked up by apollo tooling

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit a6baab3a6213abe5b8c054bb1ac517633d9b1b13

https://deploy-preview-2495--artsy-reaction.netlify.com

File: src/Artsy/SystemContext.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ export interface SystemContextProps {
    * FIXME: Ask alloy how to pass one-off props like this in from force
    */
   notificationCount?: number
+  searchQuery?: string
```
Comment (damassi) [inline src/Artsy/SystemContext.tsx]: I need to follow up this week and get proper app-level interfaces in that extend from SystemContext, and can be imported into force as a contract.

File: src/Components/NavBar/Menus/UserMenu.tsx [hunk 3]
```diff
@@ -30,8 +34,23 @@ export const UserMenu: React.FC = () => {
     })
   }
 
+  const isAdmin = user.type === "Admin"
```
Comment (pepopowitz) [inline src/Components/NavBar/Menus/UserMenu.tsx]: There's a [`userIsAdmin` helper](https://github.com/artsy/reaction/blob/master/src/Utils/user.ts#L34) that can be used to make this decision. I know it's only one line, but if some day we change the rules for what makes an admin, it'd be nice to have those decisions all made in one place.

File: src/Components/Search/SearchBar.tsx [hunk 4]
```diff
@@ -368,12 +368,18 @@ export class SearchBar extends Component<Props, State> {
 
   render() {
     return (
-      <>
+      <form
```
Comment (pepopowitz) [inline src/Components/Search/SearchBar.tsx]: 👍 🙏 ❤️

File: src/Artsy/SystemContext.tsx (hunk 2 above)
Comment (pepopowitz) [reply src/Artsy/SystemContext.tsx]: How does `searchQuery` get updated? I see it defined here, and then read [in `SearchBar.tsx`](https://github.com/artsy/reaction/pull/2495/files#diff-195cf326e4088c5f9ff2c6d47cc4248cR424), but it's not obvious to me how that value ever changes in the `SystemContext`. I'm certain it's me not understanding - can you explain?

Comment (damassi) [reply src/Artsy/SystemContext.tsx]: Yup! It's passed in from the server, in the companion PR [here](https://github.com/artsy/force/pull/4149/files#diff-3df7ffd25fef9729a61746506a33251bR39).

File: src/Components/NavBar/Menus/UserMenu.tsx (hunk 3 above)
Comment (damassi) [reply src/Components/NavBar/Menus/UserMenu.tsx]: Oh yeah, that's much nicer; will update

**Comment by artsyit** [comment]
:rocket: PR was released in v16.13.6 :rocket:

**Pull request merged**